src/cryptsetup/cryptsetup.c

```diff
@@ -136,10 +144,19 @@ static const char* const passphrase_type_table[_PASSPHRASE_TYPE_MAX] = {
         [PASSPHRASE_BOTH] = "passphrase or recovery key",
 };
 
-const char* passphrase_type_to_string(PassphraseType t);
-PassphraseType passphrase_type_from_string(const char *s);
+static const char* passphrase_type_to_string(PassphraseType t);
```

Comment:
uh, the old code was weird, these prototypes shouldn't exist at all actually, neither static nor non-static. please drop this line